guess_number/guess_word.py

```diff
@@ -0,0 +1,25 @@
+"""Pick a random word and have the user try to guess it. Provide feedback on the guessed word"""
+import random
+
+words = ["a", "be", "cat", "dog", "elf"]
+
```

Comment:
Consider wrapping all the work below in one or more well-named functions that you'll call at the end of this file to invoke it.